poc/tests/test_flp.py

```diff
@@ -265,16 +266,19 @@ def test_flp(self):
         for field in [Field64, Field128]:
             pine_valid = PineValid(field, *args)
             flp = FlpBBCGGI19(pine_valid)
-
-            # Test PINE FLP with verification.
             xof = XofTurboShake128(gen_rand(16), b"", b"")
             encoded_gradient_and_norm = \
                 flp.valid.encode_gradient_and_norm(measurement)
             (wr_check_bits, wr_check_results) = \
                 pine_valid.encode_wr_checks(encoded_gradient_and_norm[:dimension],
                                             xof)
             meas = encoded_gradient_and_norm + wr_check_bits + wr_check_results
-            test_flp_bbcggi19(flp, [(meas, True)])
+
+            for Valid in [PineValid, PineValidSquaredNormEqualityCheck]:
+                test_flp = FlpBBCGGI19(Valid(field, *args))
```

Comment:
nit: call `construct_circuits()` and iterate over the outputs:
```python
circuits = construct_circuits(BLAH)
for v in circuits:
    BLAH
```

Reply:
done